static/js/ui-utils.js

```diff
@@ -0,0 +1,422 @@
+'use strict';
+
+var openDraw = null;
+
+function rawBGsEnabled() {
+    return app.enabledOptions && app.enabledOptions.indexOf('rawbg') > -1;
+}
+
+function getBrowserSettings(storage) {
+    var json = {};
+
+    function scaleBg(bg) {
+        if (json.units == 'mmol') {
+            return Nightscout.units.mgdlToMMOL(bg);
+        } else {
+            return bg;
+        }
+    }
+
+    function appendThresholdValue(threshold) {
+        return app.alarm_types.indexOf('simple') == -1 ? '' : ' (' + scaleBg(threshold) + ')';
+    }
+
+    try {
+        var json = {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Variable 'json' is already declared in line 10.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/969293241/issues/source?bid=2452374&fileBranchId=2861014#l25)